### PR TITLE
Provide further default actions [RDY]

### PR DESCRIPTION
**Pull request by mugling**
Continues from #17058

### CODE REVIEW

File: data/json/tool_qualities.json [hunk 1]
```diff
@@ -29,7 +29,10 @@
     },{
         "type" : "tool_quality",
         "id" : "SAW_W",
-        "name" : "wood sawing"
+        "name" : "wood sawing",
+        "usages": [
+            [ 2, [ "LUMBER" ] ]
+        ]
```
Comment (Coolthulhu) [inline data/json/tool_qualities.json]: Why not keep it on one line? Short, one element arrays look clearer as one line.

**Comment by mugling** [comment]
Done.

Digging speed and labor costs now depend upon `DIG` quality

File: src/iexamine.cpp [hunk 2]
```diff
@@ -640,12 +644,12 @@ void iexamine::rubble(player &p, const tripoint &examp)
         return;
     }
 
-    // "Remove"
-    p.moves -= 200;
-    g->m.furn_set(examp, f_null);
+    // Select our best shovel
+    std::sort( shovels.begin(), shovels.end(), []( const item *lhs, const item *rhs ) {
```
Comment (Coolthulhu) [inline src/iexamine.cpp]: `std::max_element( shovels.begin(), shovels.end(), cmp )`

File: src/iuse.cpp [hunk 3]
```diff
@@ -3164,13 +3164,29 @@ int iuse::makemound(player *p, item *it, bool, const tripoint& )
...
+            // DIG 3 =  75 seconds,  2 fatigue, hunger and thirst
+            // DIG 4 =  33 seconds,  1 fatigue, hunger and thirst
+            // DIG 5 =  18 seconds,  0 fatigue, hunger and thirst
+            int bonus = std::max( it->get_quality( quality_id( "DIG" ) ) - 1, 1 );
+            bonus *= bonus;
+
+            p->moves -= 5000 / ( bonus * bonus );
+
+            if( p ) {
+                p->mod_fatigue( 10 / bonus );
```
Comment (Coolthulhu) [inline src/iuse.cpp]: Fatigue means sleepiness, not physical fatigue. Shoveling away at rubble shouldn't put you to sleep.

File: src/iexamine.cpp (hunk 2 above)
Comment (mugling) [inline src/iexamine.cpp]: Good suggestion

**Comment by mugling** [comment]
Ok, see bcdbddf

File: src/iexamine.cpp [hunk 4]
```diff
@@ -640,12 +644,12 @@ void iexamine::rubble(player &p, const tripoint &examp)
         return;
     }
 
-    // "Remove"
-    p.moves -= 200;
-    g->m.furn_set(examp, f_null);
+    // Select our best shovel
+    auto it = std::max_element( shovels.begin(), shovels.end(), []( const item *lhs, const item *rhs ) {
+        return lhs->get_quality( quality_dig ) > rhs->get_quality( quality_dig );
+    } );
```
Comment (Coolthulhu) [inline src/iexamine.cpp]: The sorting order seems to be inverted - carrying a stone shovel slows down digging immensely.

**Pull request merged**